source/cgame/blocklist.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+#define	MAX_BLOCKS	1024
+#include <stdint.h>
+typedef struct
+{
+	uint64_t steamid;
+	char name[MAX_STRING_CHARS];
```

Comment:
I'd suggest an `std::string` instead of having 1024 bytes per entry for no good reason?